### PR TITLE
Bump sphere sdk version

### PR DESCRIPTION
**Pull request by junajan**
Also added few files to gitignore and info about testing to readme

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "check-coverage": "istanbul check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
     "report-coverage": "cat ./coverage/lcov.info | codecov",
     "commit": "git-cz",
-    "test": "mocha tests/**/*.js --compilers js:babel-core/register",
+    "test": "./node_modules/.bin/mocha tests/**/*.js --compilers js:babel-core/register",
```
Comment (PhilippSpo) [inline package.json:10]: this gets automatically appended by node (?)

File: tests/integration/product-type-import.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
-import _ from 'lodash'
 import { expect } from 'chai'
 import ProductTypeImport from '../../src'
 import { SphereClient } from 'sphere-node-sdk'
 import { getSphereClientCredentials } from '../../src/utils'
 import Promise from 'bluebird'
 
-const PROJECT_KEY = 'sphere-node-product-type-import'
+const PROJECT_KEY = process.env.SPHERE_TEST_PROJECT_KEY || 'sphere-node-product-type-import'
```
Comment (PhilippSpo) [inline tests/integration/product-type-import.js:7]: 👍

**Comment by PhilippSpo** [comment]
Sadly the commit message is not `conventional-changelog` compliant. In order for this change to automatically be released and added to the changelog, you need to follow the commit message convention: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit
To make it easy you can just run `npm run commit` and follow the described steps.

File: package.json (hunk 1 above)
Comment (junajan) [reply package.json:10]: All executable scripts from _bin_ section in  package json are linked to this .bin folder
see http://www.2ality.com/2016/01/locally-installed-npm-executables.html

**Comment by junajan** [comment]
ahh.. I forgot that message format

**Pull request closed without merging**